documentation/directory-structure.md

```diff
@@ -173,75 +135,65 @@ For another example, see [`specification/eventgrid`].
 - For file names, any casing is allowed.
 - When in doubt, mimic naming of the examples provided in this article.
 
-## `specification/common-types`
-
-The special directory of [`specification/common-types`] contains shared definitions that can be reused across all Azure team services in their
-`specification` child folders.
-
-## Advanced scenario: service group
-
-In case of big Azure teams, their `specification/<azureTeam>` hosts multiple services, together known as `service group`.
-The main difference between one service and a service group is how they are presented to Azure customers:
-One service has one SDK package and one documentation portal, while a service group has separate SDK package for each service and separate documentation.
-
-For example, [`specification/containerservice`] is a `service group` for both `aks` and `fleet` services.
+## About legacy, deprecated directory structure for services and service groups
 
-The doc for `aks` is [Azure Kubernetes Service]. It points to aks REST reference e.g. for [API version `2024-01-01`][aks REST reference 2024-01-01],
-which corresponds to [`specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-01-01`].
+Many teams follow a legacy, deprecated directory structure which usually has following differences:
 
-The doc for `fleet` is [Azure Kubernetes Fleet Manager]. It point to fleet REST reference, e.g. for [API version `2023-10-15`][fleet REST reference 2023-10-15],
-which corresponds to [`specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/stable/2023-10-15`].
+- If a team owns only one service, instead of rooting it in `specification/<azureTeam>/resource-manager/<RPNS>/<service>`, it is rooted in
+  `specification/<azureTeam>/resource-manager`.
+- If a team first owned one service and then expanded to a service group, 
+  it contains mixture of both the deprecated structure (for the first service) as well as the correct structure (for the 2nd and following services).
 
-Using our example, note the most important directory structure difference of a `service group` scenario as compared to singular `service`:
+This legacy structure is not allowed going forward.
```

Comment:
Are you sure you don't want to allow this?
I think it would be OK and simplifies processes.

Reply:
@JeffreyRichter Yes. 

- First we agree to do that
- Then we will document it (I will merge this PR)
- Then we will diffuse the knowledge to all PR reviewers, (ARM, SDK gen, etc.)
- Then we will enforce it in tooling. We have some work todo this properly (details [in this comment](https://github.com/Azure/azure-rest-api-specs/issues/29065#issuecomment-2104104232)) but I think we could quickly deploy some simple check validating just that.

@mikeharder does that sound good to you too?

Reply:
I synced with @mikeharder and we agreed. Details here:

- https://github.com/Azure/azure-rest-api-specs/issues/29065#issuecomment-2148357551

Reply:
We chatted in person. Meeting notes here:
- https://github.com/Azure/azure-rest-api-specs/pull/28929#issuecomment-2151095434